packages/dds/merge-tree/src/test/mergeTreeOperationRunner.ts

```diff
@@ -360,11 +361,24 @@ export function generateOperationMessagesForClients(
 		let op: IMergeTreeOp | undefined;
 		if (len === 0 || len < minLength) {
 			const text = client.longClientId!.repeat(random.integer(1, 3));
-			op = client.insertTextLocal(random.integer(0, len), text);
+			let pos = random.integer(0, len);
```

Comment:
i'd prefer if the field concept didn't leak out of the obliterate farm, and this logic didn't really chang
. i'd do something like:
1. add a new parameter to the function, `insertText`
2. put the existing code to insert in a function in this file, and set the new parameter to default to that.
3. pass a field knowledgeable version in the obliterate farm